scripts/gpu2ranks_ompi

```diff
@@ -26,4 +26,4 @@ then
   fi
 fi
 
-eval "$@"
+"$@"
```

Comment:
I would remove this file completely, as it is not used anymore.

Reply:
Good point. Removed in https://github.com/eth-cscs/DLA-Future/pull/1093/commits/a44eacc6eba3a1424179f784951467c99590dc91.